kuma/static/styles/components/compat-tables/bc-icons.scss

```diff
@@ -4,17 +4,26 @@
 .bc-browsers {
     .bc-head-txt-label {
         display: inline-block;
-        margin: 0 0 0 50px;
+        bottom: 0;
+        margin: 0 0 20px 10px;
         padding: 0;
-        width: 140px;
-        height: 210px;
+        width: 20px;
+        height: 20px;
         transform: rotate(-90deg);
+        transform-origin: left;
         @include set-font-size($small-bump-font-size);
         text-align: left;
         white-space: nowrap;
 
-        @media #{$mq-tablet-only} {
-            margin-left: 42px;
+        /* At this breakpoint the table uses grid layout
+           where supported. This stops `vertical-align: bottom`
+           from working so, we need to switch to this workaround
+           to ensure the layout still works. */
+        @supports (display: grid) {
+            @media #{$mq-tablet-small-desktop-only} {
+                position: relative;
+                bottom: -150px;
+            }
```

Comment:
Wouldn't these properties be better in the bc-table-tablet-display-grid mixin in bc-table.scss?

Thanks for the detailed comment here. But I notice that it refers to a vertical-align property that appears in bc-table.scss, which also makes me think that maybe this special case should be in that file. bc-table.scss already has one mention of the bc-browsers class in it, so it wouldn't be breaking abstractions too badly to move this there, I suspect.

Having said all that, I should confess that I'm not very familiar with scss, so don't know what best practices are.